etna/libs/ts2vec/ts2vec.py

```diff
@@ -333,5 +341,5 @@ def load(self, fn):
         Args:
             fn (str): filename.
         '''
-        state_dict = torch.load(fn, map_location=self.device)
+        state_dict = torch.load(fn, map_location=torch.device("cpu"))
```

Comment:
Why do we use cpu here?

Reply:
I think it is better when model is loading to cpu in the beginning.
There is no need to be on gpu when model is not used.

Reply:
Shouldn't we clarify this behavior somewhere in the docs? It isn't obvious.